plugins/qlcplus.js

```diff
@@ -51,24 +52,39 @@ function exportAddChannel(xml, channel, fixture) {
     }
   });
 
-  const dataChannel = getDataChannel(channel, fixture);
-  const chType = getChannelType(dataChannel);
+  // use default channel's data
+  channel = channel instanceof SwitchingChannel ? fixture.getChannelByKey(channel.defaultChannelKey) : channel;
```

Comment:
Better use an if statement here since the else case is just leaving it as it is.